tests/CacheCompat/CommonCache.Test.Unit/CommonCache.Test.Unit.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.9" />
```

Comment:
It looks like you changed this package ref name, not just the version.

Reply:
Reverting these changes.